apps/change-log.md

```diff
@@ -14,6 +14,28 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2023-04-13
+
+### Changes with software release 7.82.0
+
+#### <i class="fa fa-plus"></i> New
+
+* [`POST`/products/find](https://developer.epages.com/apps/api-reference/post-shopid-products-find.html)
+* [`GET`/producttypes](https://developer.epages.com/apps/api-reference/get-shopid-productytypes.html)
+* [`POST`/producttypes](https://developer.epages.com/apps/api-reference/post-shopid-productytypes.html)
+* [`GET`/producttypes/{productTypeId}](https://developer.epages.com/apps/api-reference/get-shopid-productytypes-producttypeid.html)
```

Comment:
I get a 404 page for these 3 links - looks like you have a typo in there: "productytypes" should be "producttypes" :)

Reply:
@ywolpers Your are completely right. Good catch! Thanks!